.github/workflows/release.yaml

```diff
@@ -333,3 +333,42 @@ jobs:
       base64-subjects: "${{ needs.combine_hashes.outputs.hashes }}"
       upload-assets: true # Optional: Upload to a new release
       provenance-name: kargo-cli.intoto.jsonl
+
+  update-homebrew-formula:
+    if: github.event_name == 'release'
+    permissions:
+      contents: write # Needed to push changes
+      pull-requests: write # Needed to create a pull request
```

Comment:
Something to look out for. I _think_ these are permissions WRT to the current repository. This may not be sufficient to open a PR in a _different_ repository. We'll find out when we try this.

Reply:
You are correct, for permissions to another repository you need to use a token that has access to this repository. This also means we do not have to elevate the permissions for the step here.

Reply:
The tap absolutely needs to be a separate repo. Having played with this quite a bit recently, I'm pretty confident that things work most seamlessly when the tap repo is named <org>/homebrew-tap.

(Semi-related: akuity/homebrew-tap is not currently following the conventional layout for a tap, but it does work and we can deal with that separately later.)

Here, you can use `secrets.TAP_PAT` and I'll ensure that secret is configured with a PAT that can write to akuity/homebrew-tap.